.github/workflows/newsletter-issue.yml

```diff
@@ -27,8 +29,8 @@ jobs:
       - name: IRC notification
         uses: Gottox/irc-message-action@v1
         with:
-          channel: '#pythonpune'
-          nickname: pypune_gh_${{ github.run_id }}
+          channel: ${{ secrets.channel }}
```

Comment:
What is this secret identifer?

Reply:
This might help, https://docs.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets